kuma/static/styles/components-skinny/wiki/communitybox.scss

```diff
@@ -0,0 +1,81 @@
+/*
+  blue box on footer of many landing pages
+  Generated by macros:
+    https://github.com/mdn/kumascript/blob/master/macros/TopicBox.ejs
+    https://github.com/mdn/kumascript/blob/master/macros/CommunityBox.ejs
+********************************************************************** */
+
+$communitybox-spacing : 13px;
+
+.communitybox {
+    padding: $communitybox-spacing;
+    background-color: $blue-light;
+    @include bidi-value(border, 0, 0);
+    @include set-font-size($body-font-size);
+
+    strong {
+        @include set-larger-font-size();
+    }
+
+    ul {
+        padding: 0;
+        margin-bottom: 0;
+        list-style-type: none;
+    }
+
+    h2 {
+        margin-bottom: 0;
+    }
+
+    h3 {
+        font-style: normal;
+    }
+}
+
+/* subheading */
+.communitysubhead {
+    @include set-larger-font-size();
+    font-weight: bold;
+
+    /* only need margin to balance it with content in other columns */
+    .column-half & {
+        margin-top: $communitybox-spacing;
+        margin-bottom: $content-vertical-spacing;
+    }
+}
+
+/* links to mailing list */
+ul.communitymailinglist { /* needs specificity to get past .text-content ul */
+    margin: $communitybox-spacing 0 ($content-vertical-spacing * -1) 0;
```

Comment:
Nit: This can be shortened to `margin: $communitybox-spacing 0 ($content-vertical-spacing * -1);`?